code/game/machinery/computer/orders/order_items/mining/order_consumables.dm

```diff
@@ -0,0 +1,42 @@
+/datum/orderable_item/consumables
+	category_index = CATEGORY_CONSUMABLES
+
+/datum/orderable_item/consumables/survival_pen
+	item_path = /obj/item/reagent_containers/hypospray/medipen/survival
+	cost_per_order = 500
+
+/datum/orderable_item/consumables/luxury_pen
+	item_path = /obj/item/reagent_containers/hypospray/medipen/survival/luxury
+	cost_per_order = 1000
+
+/datum/orderable_item/consumables/medkit
+	item_path = /obj/item/storage/medkit/brute
+	cost_per_order = 600
+
+/datum/orderable_item/consumables/whiskey
+	item_path = /obj/item/reagent_containers/cup/glass/bottle/whiskey
+	cost_per_order = 100
+
+/datum/orderable_item/consumables/absinthe
+	item_path = /obj/item/reagent_containers/cup/glass/bottle/absinthe/premium
+	cost_per_order = 100
+
+/datum/orderable_item/consumables/bubblegum
+	item_path = /obj/item/storage/box/gum/bubblegum
+	cost_per_order = 100
+
+/datum/orderable_item/consumables/havana_cigars
+	item_path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost_per_order = 150
+
+/datum/orderable_item/consumables/havana_cigars
+	item_path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost_per_order = 150
+
+/datum/orderable_item/consumables/tracking_implants
+	item_path = /obj/item/storage/box/minertracker
+	cost_per_order = 600
+
+/datum/orderable_item/consumables/space_cash
```

Comment:
Why is space cash orderable via cargo shuttle but only via mining?

Reply:
All the items were just ported from the mining vendor. I think it's just meant to say 2000mining points = 1000cr

Reply:
No, I knew what it was, but what I'm saying is that maybe this is a bit silly to retain this as a feature, because why would you be shipping money via the cargo shuttle that isn't in a personal envelope? Open cash isn't really a trade good.